proto/serverless/proto/instrumentation/v1/log.proto

```diff
@@ -0,0 +1,54 @@
+syntax = "proto3";
+
+package serverless.proto.instrumentation.v1;
+
+import "serverless/proto/common/v1/common.proto";
+
+// A LogPayload is a message that will contain any number of
+// LogEvents plus the global tags required by our Serverless Ingest Platform.
+message LogPayload {
+    serverless.proto.common.v1.SlsTags sls_tags = 1;
+
+    // A list of LogEvents to be ingested. Ingest does not impose a limit on the
+    // number of LogEvents in a single payload. It is the responsibility of
+    // the LogEvents' producers to limit the size of payloads based on their
+    // own requirements.
+    repeated LogEvent log_events = 2;
+}
+
+message LogEvent {
```

Comment:
We will need tags attached to log events so we can filter logs in dev mode

Reply:
Where do these context properties come from? Curious if we will have a gap here. Also what tags should be attached? Should we just attach the standard `Tags` message to all logs?

Reply:
Right now the only filters we tags we allow customer to filter by are `environment` and `namespace` so we could just keep it limited to those and add more as we add new filters?

Actually you know what, ignore me about the context properties. I just double checked and forgot that those properties were added post instrumentation 👍